sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1457,6 +1462,24 @@ public void setTraceSampling(boolean traceSampling) {
     this.traceSampling = traceSampling;
   }
 
+  /**
+   * Returns a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
+   *
+   * @return the list of origins
+   */
+  public @NotNull List<String> getTracingOrigins() {
+    return tracingOrigins;
+  }
+
+  /**
+   * Sets a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
+   *
+   * @param tracingOrigins - the tracing origins
+   */
+  public void setTracingOrigins(final @NotNull List<String> tracingOrigins) {
+    this.tracingOrigins = tracingOrigins;
+  }
```

Comment:
usually, we offer the methods `addX` eg `addEventProcessor` but never the instance of the List itself